UnitTestBot/UTBotJava
# plugin.xml changes for release 2023.10

## Description

plugin.xml changes:
1. Minor Overview correction
2. New What's New text

## How to test

### Manual tests

1. Publish IntelliJ plugin
2. Install into IDEA 2023.2
3. Check Overview and What's New tabs in IDEA (Settings->Plugins->Installed)

## Self-check list
